packages/loader/container-loader/src/container.ts

```diff
@@ -2586,3 +2587,17 @@ export interface IContainerExperimental extends IContainer {
 	 */
 	closeAndGetPendingLocalState?(stopBlobAttachingSignal?: AbortSignal): Promise<string>;
 }
+
+/**
+ * @deprecated Will be removed in future major release.
+ * @alpha
+ */
+export async function requestOnIContainer_Deprecated(container: IContainer): Promise<FluidObject> {
+	const containerCast = container as Container;
+	if ("request" in containerCast.runtime && typeof containerCast.runtime.request === "function") {
+		const response: IResponse = await (containerCast.runtime as any).request({ url: "/" });
```

Comment:
Why is the cast of `runtime as any` necessary?
Also, to the extent that it relies on the `runtime.request` method that is slated to be removed -- and especially to the extent that there isn't a typing guarantee -- do you want to add a note on [the containerRuntime.ts file](https://github.com/microsoft/FluidFramework/blob/262c4be092dd4d44c23bed9675728152b1d587d5/packages/runtime/container-runtime/src/containerRuntime.ts#L1745C1-L1751C64) that the method can't be removed for some time?